week-2/M-array-map/exercise.js

```diff
@@ -6,11 +6,11 @@
 */
 
 function double(num) {
-  return num * 2;
+  return numbersDoubled;
```

Comment:
I think the exercise actually wanted you to `return num * 2;` in here and call
`numbers.map(double);`
- however, you've gone to a slightly more advanced way of doing it, which is good as well.